x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/method_connector/method_connector.tsx

```diff
@@ -180,10 +198,10 @@ export const MethodConnector: React.FC = () => {
               <EuiText size="s">
                 <p>
                   {i18n.translate(
-                    'xpack.enterpriseSearch.content.newIndex.steps.buildSearchExperience.content',
+                    'xpack.enterpriseSearch.content.newIndex.connector.steps.buildSearchExperience.content',
                     {
                       defaultMessage:
-                        'Connect your newly created Elasticsearch index to an App Search engine to build a cusomtizable search experience.',
+                        'After building your connector, your content is ready. Build your first search experience with Elasticsearch, or explore the search experience tools provided by App Search and Workplace Search. We recommend that you create a search engine for the best balance of flexible power and turnkey simplicity.',
```

Comment:
I don't think that you can _use_ workplace search in any capacity with a connector-index-in-content-plugin?

Reply:
Nice catch, removing that.